python/pylibcudf/pylibcudf/nvtext/tokenize.pyx

```diff
@@ -0,0 +1,286 @@
+    """
+    Returns a single column of strings by tokenizing the input
+    strings column using the provided characters as delimiters.
+
+    For details, see cpp:func:`cudf::nvtext::tokenize`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to tokenize
+    delimiter : Scalar
+        String scalar used to separate individual
+        strings into tokens
+
+    Returns
+    -------
+    Column
+        New strings columns of tokens
+    """
+    cdef unique_ptr[column] c_result
+
+    if delimiter is None:
+        delimiter = Scalar.from_libcudf(
+            cpp_make_string_scalar("".encode())
+        )
+
+    with nogil:
+        c_result = cpp_tokenize(
+            input.view(),
+            dereference(<const string_scalar*>delimiter.c_obj.get()),
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column tokenize_column(Column input, Column delimiters):
+    """
+    Returns a single column of strings by tokenizing the input
+    strings column using multiple strings as delimiters.
+
+    For details, see cpp:func:`cudf::nvtext::tokenize`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to tokenize
+    delimiters : Column
+        Strings column used to separate individual strings into tokens
+
+    Returns
+    -------
+    Column
+        New strings columns of tokens
+    """
+    cdef unique_ptr[column] c_result
+
+    with nogil:
+        c_result = cpp_tokenize(
+            input.view(),
+            delimiters.view(),
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column count_tokens_scalar(Column input, Scalar delimiter=None):
+    """
+    Returns the number of tokens in each string of a strings column
+    using the provided characters as delimiters.
+
+    For details, see cpp:func:`cudf::nvtext::count_tokens`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to count tokens
+    delimiters : Scalar]
+        String scalar used to separate each string into tokens
+
+    Returns
+    -------
+    Column
+        New column of token counts
+    """
+    cdef unique_ptr[column] c_result
+
+    if delimiter is None:
+        delimiter = Scalar.from_libcudf(
+            cpp_make_string_scalar("".encode())
+        )
+
+    with nogil:
+        c_result = cpp_count_tokens(
+            input.view(),
+            dereference(<const string_scalar*>delimiter.c_obj.get()),
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column count_tokens_column(Column input, Column delimiters):
+    """
+    Returns the number of tokens in each string of a strings column
+    using multiple strings as delimiters.
+
+    For details, see cpp:func:`cudf::nvtext::count_tokens`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to count tokens
+    delimiters : Column
+        Strings column used to separate
+        each string into tokens
+
```

Comment:
The line widths on some of these docstrings are oddly short. Any idea what's up?